cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -502,7 +502,7 @@ internal class ServiceImpl(
                 .executeOnLeft { error ->
                     (error as? ConsentLibExceptionK)?.let { logger.error(error) }
                     val spConsents = ConsentManager.responseConsentHandler(
-                        gdpr = campaignManager.gdprConsentStatus?.copy(applies = dataStorage.gdprApplies),
+                        ccpa = campaignManager.ccpaConsentStatus?.copy(applies = dataStorage.ccpaApplies),
```

Comment:
I think that was a typo

Reply:
Good catch. Could've been a copy & paste mistake.